.github/workflows/check-go-dependencies-task.yml

```diff
@@ -43,7 +43,7 @@ jobs:
       - name: Determine if the rest of the workflow should run
         id: determination
         run: |
-          RELEASE_BRANCH_REGEX="refs/heads/[0-9]+.[0-9]+.x"
+          RELEASE_BRANCH_REGEX="refs/heads/v[0-9]+.[0-9]+.x"
```

Comment:
See https://github.com/arduino/arduino-cli/pull/2374#discussion_r1363729420

Reply:
Without the suggested change, if the PR is merged before the first "v" format tag is pushed then it will immediately break installation from the script between the time of the merge and when the first "v" format tag is pushed:

```text
$ curl -fsSL https://raw.githubusercontent.com/umbynos/arduino-cli/new-v-format/install.sh | sh
Installing in /e/stuff/todo/bin
ARCH=64bit
OS=Windows
Using curl as download tool
Failed to install arduino-cli
```

Reply:
Yeah I think no one will install the cli in that short period of time, but maybe it's better to stay cautious, I'll add this back

Reply:
Hundreds of 3rd party repositories use the script in their CI system, so there is quite a lot of installation activity:

https://github.com/adafruit/ci-arduino/blob/master/actions_install.sh#L32

Reply:
This is causing errors, please see: https://github.com/arduino/arduino-cli/issues/2370